Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -156,7 +156,7 @@ private void launchPayPal(
                 buyerEmailAddress,
                 buyerPhoneCountryCode,
                 buyerPhoneNationalNumber,
-                    "fake-session-id"
```

Comment:
It's just a stage problem that this isn't working? I would expect this field to be passable. We should make sure to put this back and test on prod before release.